README.md

```diff
@@ -1,4 +1,4 @@
-roslibjs [![Build Status](https://api.travis-ci.org/RobotWebTools/roslibjs.png)](https://travis-ci.org/RobotWebTools/roslibjs)
```

Comment:
Maybe I have done some mistake, but the readme file can remain as the original one